classes/class-settings.php

```diff
@@ -817,13 +817,14 @@ public function render_field( $field ) {
 					// Author or Role dropdown menu
 					$author_or_role_values   = array();
 					$author_or_role_selected = array();
+					$users = count_users();
```

Comment:
@frozzare You can move this even higher up, outside the `foreach` of the `exclude_row`. Perhaps place it at the beginning of the `rule_list` case.